content/en/docs/concepts/traffic-management/index.md

```diff
@@ -13,237 +13,155 @@ aliases:
-and clients sending requests don't distinguish between different versions of
-the service.
-
-With Istio, because the Envoy proxy intercepts and forwards all requests and
-responses between the clients and the services, you can use routing rules with
-[service subsets](/docs/concepts/traffic-management/#service-subsets)
-in a virtual service to configure the [routing rules](/docs/concepts/traffic-management/#routing-rules)
-for multiple versions of a service.
-
-Service subsets are used to label all instances that correspond to a specific
-version of a service.
-Before you configure routing rules, the Envoy proxies use round-robin load
-balancing across all service instances, regardless of their subset. After you
-configure routing rules for traffic to reach specific subsets, the Envoy
-proxies route traffic to the subset according to the rule but again use
-round-robin to route traffic across the instances of each subset.
-
-This configuration method provides the following advantages:
-
-- Decouples the application code from the evolution of the application's dependent services.
-- Provides monitoring benefits.
-For details, see [Mixer policies and telemetry](/docs/reference/config/policy-and-telemetry/).
-
-For example, in A/B testing we often want to configure traffic routes based on
-percentages. With Istio, you can use a virtual service to specify a routing
-rule that sends 25% of requests to instances in the `v2` subset, and sends the
-remaining 75% of requests to instances in the `v1` subset. The following
-configuration accomplishes our example for the `reviews` service.
+-   [Introducing Istio Traffic Management](/docs/concepts/traffic-management/#introducing-istio-traffic-management):
+    Learn about how Istio helps you control the flow of traffic in your mesh
+    with simple, flexible configuration.
+
+-   [Network resilience and testing](/docs/concepts/traffic-management/#network-resilience-and-testing):
+    Learn about Istio's dynamic failure recovery features that you can configure
+    to test and build tolerance for failing nodes, and to prevent cascading
+    failures to other nodes.
+
+-   [Architecture](/docs/concepts/traffic-management/#architecture): Learn about
+    Pilot, Istio's core traffic management component and Envoy proxies and how
+    they enable service discovery and traffic control for services in the mesh.
+
+Istio’s traffic routing rules let you easily control the flow
+of traffic and API calls between services. Istio simplifies configuration of
+service-level properties like circuit breakers, timeouts, and retries, and makes
+it easy to set up important tasks like A/B testing, canary rollouts, and staged
+rollouts with percentage-based traffic splits. It also provides out-of-box
+failure recovery features, which, combined with Istio’s
+[observability](/docs/concepts/observability/) features, helps make your calls
+more reliable, and your network more robust.
```

Comment:
helps make your application more robust against failures of dependent services or the network